cooked-validators/src/Cooked/Tx/Constraints/Type.hs

```diff
@@ -665,14 +907,12 @@ data TxSkel where
       txSkelMints :: TxSkelMints,
       txSkelValidityRange :: Pl.POSIXTimeRange,
       txSkelRequiredSigners :: Set Pl.PubKeyHash,
-      txSkelIns :: Map SpendableOut TxSkelIn,
+      txSkelIns :: Map Pl.TxOutRef TxSkelRedeemer,
       txSkelOuts :: [TxSkelOut],
```

Comment:
Is there a mechanism somewhere to make sure that the generated transaction keeps the same order for the outputs specified here? We used to have this hack where we would gather the generated transaction from p-apps and sort it properly based on the order of our contraints, but I assume, this is more naturally embedded in the transaction function now.